src/game.cpp

```diff
@@ -3725,7 +3725,8 @@ std::vector<std::string> game::list_active_characters()
  */
 void game::write_memorial_file(std::string sLastWords)
 {
-    const std::string &memorial_dir = FILENAMES["memorialdir"];
+    const std::string &memorial_dir = FILENAMES["memorialdir"] +  world_generator->active_world->world_name + "/";
```

Comment:
I don't know. It worked here, but I'm running win64. I echoed the world_generator->active_world->world_path string, and that had unix path separators. So I assumed it would work.